view/frontend/email/tesco_template.html

```diff
@@ -7,7 +7,7 @@
         Lotus store within the next 24 hours to finalise payment. After that time your order will be automatically
```

Comment:
I suppose if we have "Valid until" below, there's not really any value to writing it out in words up here, too, lest the expiration period changes at some point.  Just want to remove any potential for future inconsistencies.

Reply:
Should we update this line of template? 
"Please print it and bring it to any Tesco Lotus store within valid time to finalise payment" or any better suggestion?

Reply:
How about:

> Please print it and bring it to any Tesco Lotus store while the charge is still valid to finalize payment

(note the American spelling of "finalize")